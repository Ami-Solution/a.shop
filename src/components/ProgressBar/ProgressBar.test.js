/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ProgressBar from './index';

configure({ adapter: new Adapter() });

describe('Component ProgressBar', () => {
  const component = renderer
    .create(<ProgressBar progressRatio={1/5} />);
  const tree = component.toJSON();

  it('should be render without crash', () => {
    expect(tree).toMatchSnapshot();
  });
});
