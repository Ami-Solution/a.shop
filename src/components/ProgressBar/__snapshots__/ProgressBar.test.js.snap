// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ProgressBar should be render without crash 1`] = `
<div
  className="sc-bdVaJa lgyZwb"
>
  <div
    className="sc-htpNat dkNOeq"
  >
    <div
      className="sc-bxivhb eEDTpv"
    />
  </div>
</div>
`;
